packages/kit/src/utils/import.js

```diff
@@ -9,8 +7,7 @@ import { pathToFileURL } from 'node:url';
  */
 export async function resolve_peer_dependency(dependency) {
 	try {
-		// @ts-expect-error the types are wrong
-		const resolved = imr.resolve(dependency, pathToFileURL(process.cwd() + '/dummy.js'));
+		const resolved = resolveModuleURL(dependency);
```

Comment:
we could use `{ try: true }` here also to avoid try/catch situation, however I didn't want to make too much changes.